### DISCUSSION TITLE
Renaming nodes

### DISCUSSION THREAD

**Discussion by maukujnr** [vitobotta/hetzner-k3s#158] [Q&A]
How do I get to configure how to name my master and worker nodes? for instance clustername-worker-01 or clustername-master-01

**Comment by vitobotta** [comment]
It's not possible currently and it would make things a lot more complicated. The tool needs to be aware of a consistent naming scheme to be able to operate on an existing cluster when you make changes. Custom names would make this more difficult. Generally speaking, with the way Kubernetes works you shouldn't really need to care about such details IMO. What is the reason you would like to customize the names?
